dist/yaml/ovnkube-master.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         component: network
         type: infra
         openshift.io/component: network
+        os.kubernetes.io='linux'
```

Comment:
This should be:
```
beta.kubernetes.io/os: linux
```

Reply:
And it should be in the nodeSelector arg.

Reply:
changed